lib/experimental/media/load.php

```diff
@@ -186,6 +186,24 @@ function gutenberg_rest_get_attachment_filesize( array $post ): ?int {
 	return null;
 }
 
+/**
+ * Filters the list of rewrite rules formatted for output to an .htaccess file.
+ *
+ * Adds support for serving wasm-vips locally.
+ *
+ * @param string $rules mod_rewrite Rewrite rules formatted for .htaccess.
+ * @return string Filtered rewrite rules.
+ */
+function gutenberg_filter_mod_rewrite_rules( string $rules ): string {
+	$rules .= "\n# BEGIN Gutenberg client-side media processing experiment\n" .
+				"AddType application/wasm wasm\n" .
+				"# END Gutenberg client-side media processing experiment\n";
+
+	return $rules;
+}
+
+add_filter( 'mod_rewrite_rules', 'gutenberg_filter_mod_rewrite_rules' );
```

Comment:
Does filtering this automatically trigger a Core update of rewrite rules? I'm not too familiar with this filter, but won't it need `flush_rewrite_rules()` or something similar to put this into `.htaccess`?

Reply:
Yes it does, but that felt a bit out of scope for this PR. Maybe in a new PR I could add a flush every time the option is changed or so.